java/ql/src/Security/CWE/CWE-094/SSTIBad.java

```diff
@@ -0,0 +1,42 @@
+import javax.servlet.http.HttpServletRequest;
```

Comment:
Abbreviate examples to a minimal snippet -- we don't need to quote the imports, for example, unless there's a really unclear type name (here `Velocity` and `VelcityContext` are both obvious)